proposals/restructure-KF-docs-proposal.md

```diff
@@ -0,0 +1,232 @@
+This is an initial proposal, to be iterated on, following the discussion in [#2293](https://github.com/kubeflow/website/issues/2293).
+
+### Kubeflow documentation structure
+
+- **Getting started**
+  - Introduction (current About/Kubeflow)
+  - Concepts (current Kubeflow Overview)
+  - Use Cases (current About/Use Cases)
+  - Install Kubeflow (current `Installing Kubeflow`, to redirect to `Deployment` homepage)
+  - [delete] Cloud Installation
+    
+- **Community**
```

Comment:
Does `Community` need to be so high up the top-level navigation? As a user of the KF docs for the last year I've used the components docs a bunch more and whilst I definitely agree it's important to put `Community` in the top-level navigation putting it closer to the bottom of the list makes sense to me.

Reply:
+1

Reply:
Solved!